### ISSUE TITLE
Question: can it decode also time-compressed representations?

### ISSUE THREAD

**Issue by TamirEliav** [CYHSM/DeepInsight#8]
Hi Markus,
I really liked this paper, and want to give it a try with other datasets.
I have two questions:
1. Can it decode time-compressed representations, like replay in the hippocampus? assuming a constant compression factor (but maybe unknown value). If not, do you think the model can be expended to handle it?
2. How sensitive is it to (slow) drifts in the neural activity? i.e. slow changes in the spikes amplitude.
Thanks a lot! Tamir :)

**Comment by CYHSM** [comment]
Thanks a lot, let me see if I can answer some of these questions. 

1. As of now, there is no quantification of replay implemented in DeepInsight, but let me try to speculate how it could be captured. In theory like any other decoder, DeepInsight will try to capture spatially robust signals from the training set to minimize the distance loss. This means a replay event happening in the test set can be captured by looking at the predicted positions, most likely the ones with a high error which are temporally and spatially constrained. This, of course, only works if the model did not already account for the replay events in the training set, which should generally be true as the number of replay events is (probably) smaller than the number of place field crossings given a large enough training window. 

2. Good question, I think weight sharing across channels can help find a more general filter in the convolutional layers which can robustly pick up spikes independent of how the exact spike signal looks like. The question remains if the model internally uses some spike clustering to make sure if it is keeping track of the cells, which I am unsure about (comparison to a 3DConv model without weight sharing might be used to answer this).
